modules/ROOT/pages/migrate-mule3-apimanager.adoc

```diff
@@ -7,19 +7,19 @@ You migrate your existing Mule 3 applications that are managed by API Manager to
 
 . <<prepare-migration,Prepare for your migration>>
 +
-Based on how your existing Mule 3 application is configured, carefully review the prerequisites and determine how to plan the migration.
+Based on how your existing Mule 3 application is configured, carefully review the prerequisites to prepare for the migration.
 . <<determine-migration-approach,Determine the migration approach>>
 +
-Based on several factors, such as whether the migration requires a downtime, requires removing applied policies, or requires migrating contracts, you use the existing API instance or create an instance for migration.
+Based on several factors, such as whether the migration requires a downtime, requires removing applied policies, or requires migrating contracts, you use an existing API instance or create an instance for the migration.
 . <<perform-migration,Perform the migration>>
 +
-Based on whether you want to create an API or use an existing one, you can appropriately migrate your application.
+Migrate your application using an existing or a new API.
```

Comment:
Given Line 13, consider shortening to:
```suggestion
Migrate your application.
```
OR simply delete this in favor of the bullet text alone.